ts/parts/Chart.ts

```diff
@@ -144,6 +144,7 @@ declare global {
             public getContainer(): void;
             public getMargins(skipAxes?: boolean): void;
             public getSelectedPoints(): Array<Point>;
+            public getSelectedParentNodes(): Array<PackedBubblePoint>;
```

Comment:
Don't set it in `Chart.ts`. Instead extend Chart in the file that creates method. For an example check `BubbleLegend.ts`.